packages/@webex/plugin-meetings/src/meetings/util.ts

```diff
@@ -59,6 +59,23 @@ MeetingsUtil.handleRoapMercury = (envelope, meetingCollection) => {
         };
 
         meeting.mediaProperties.webrtcMediaConnection.roapMessageReceived(roapMessage);
+
+        let mediaServer;
+
+        // Attempt to collect the media server from the roap message.
+        try {
+          mediaServer = roapMessage.sdp
+            .split('\r\n')
+            .find((line) => line.startsWith('o='))
+            .split(' ')
+            .shift()
+            .replace('o=', '');
+        } catch {
+          mediaServer = undefined;
+        }
+
+        meeting.mediaProperties.webrtcMediaConnection.mediaServer = mediaServer;
```

Comment:
`sdp` is undefined for most roap messages, so you end up with the catch and overwriting `meeting.mediaProperties.webrtcMediaConnection.mediaServer` with `undefined`. It's better to check roapMessage.messageType to be ROAP.ROAP_TYPES.ANSWER or ROAP.ROAP_TYPES.OFFER